static/sass/styles-v1.scss

```diff
@@ -176,3 +178,9 @@ fieldset {
   font-size: 0.875rem;
   text-transform: uppercase;
 }
+
+// XXX p-list-step__content with code blocks
```

Comment:
This should be addressed upstream? Can you please create an issue and add the link to this comment?